go/filemanager/pfsmodules/chunkmeta.go

```diff
@@ -30,10 +28,16 @@ type ChunkMeta struct {
 	Len      int64  `json:"len"`
 }
 
+// ToString  pack a info tring of ChunkMeta.
+func (m *ChunkMeta) ToString() string {
```

Comment:
Not a requirement, but could consider using the `Stringer` interface here: https://tour.golang.org/methods/17 .